src/evaluations/data/evaluation_configs.py

```diff
@@ -14,14 +14,21 @@
 """Evaluation configurations."""
 import numpy as np
 
+from absl import flags
+
 from wfa_cardinality_estimation_evaluation_framework.estimators import bloom_filters
+from wfa_cardinality_estimation_evaluation_framework.estimators import exact_set
 from wfa_cardinality_estimation_evaluation_framework.estimators import liquid_legions
 from wfa_cardinality_estimation_evaluation_framework.estimators import vector_of_counts
 from wfa_cardinality_estimation_evaluation_framework.evaluations.configs import EvaluationConfig
 from wfa_cardinality_estimation_evaluation_framework.evaluations.configs import ScenarioConfig
 from wfa_cardinality_estimation_evaluation_framework.simulations import set_generator
 from wfa_cardinality_estimation_evaluation_framework.simulations.simulator import SketchEstimatorConfig
 
+FLAGS = flags.FLAGS
+
+flags.DEFINE_integer('max_frequency', 10, 'Maximum frequency to be analyzed.')
```

Comment:
I'd prefer if we avoid declaring flags in files that do not have main functions.